themes/src/main/resources/theme/base/login/messages/messages_en.properties

```diff
@@ -14,6 +14,7 @@ doClickHere=Click here
 doImpersonate=Impersonate
 doTryAgain=Try again
 doTryAnotherWay=Try Another Way
+doConfirmDelete=Confirm deletion
```

Comment:
@andymunro  can you review these changes? Also see email "KEYCLOAK-953: allow user to delete account" in keycloak-dev ML for screenshots.